src/BuildingRegistry.Producer.Snapshot.Oslo/ProducerBuildingUnitProjections.cs

```diff
@@ -500,6 +514,25 @@ public ProducerBuildingUnitProjections(IProducer producer, ISnapshotManager snap
                         ct);
                 }
             });
+
+            When<Be.Vlaanderen.Basisregisters.ProjectionHandling.SqlStreamStore.Envelope<BuildingUnitWasTransferred>>(async (_, message, ct) =>
+            {
+                await FindAndProduce(async () =>
+                        await snapshotManager.FindMatchingSnapshot(
+                            message.Message.BuildingUnitPersistentLocalId.ToString(),
+                            message.Message.Provenance.Timestamp,
+                            message.Message.GetHash(),
+                            message.Position,
+                            throwStaleWhenGone: false,
+                            ct),
+                    message.Position,
+                    ct);
+            });
+
+            When<Be.Vlaanderen.Basisregisters.ProjectionHandling.SqlStreamStore.Envelope<BuildingUnitWasMoved>>(async (_, message, ct) =>
+            {
+                await Produce($"{osloNamespace}/{message.Message.BuildingUnitPersistentLocalId}", "{}", message.Position, ct);
```

Comment:
This {} doesn't look correct. By doing this, you delete the building unit. But it keeps existing, just below another building (id).